hcert_spec.md

```diff
@@ -217,23 +217,21 @@ A proposed payload schema for [EU Health Certficate v1](https://github.com/ehn-d
+The Member State certificates are called Certificate Signer Certificate Authorities (CSCAs) and are (typically) self-signed certificates. Countries may have more than one (e.g. in case of regional devolution). These CSCA certificates regularly sign the Document Signing Certificates (DSCs) used for signing HCERTs. Member States will each maintain a public register of the DSC certificates that is kept current and published at a stable URL which is communicated to the Secretariat. Member States MUST remove any revoked or stale certificates from this list.
 
-These CSCA certificates regularly sign the Document Signing Certificates (DSC) used in day to day operations. Memberstates will each will maintain a public register of the DSC certificates that is kept current at a stable URL communicated to the secretariat.  Memberstates *must* remove any revoked certificates from this list.
+The Secretariat will regularly retrieve the Member States DSCs, verify these agains the list of CSCA certificates (which have been conveyed and verified by other means) and publish an aggregated list of DSCs in various formats for Verifiers to retrieve. 
 
-Other memberstates must regularly fetch these list of DSC certificates and cryptographically verify these against the CSCA certificates (that they have verified by other, non-digital, means). Such fetch should only contain the certificates that that memberstates considers valid at the time of the fetch. 
-
-Verifiers should fetch and update this list regularly. This list of DSC certificates then provides the basis for the acceptable public keys (and the corresponding KIDs) that Verifiers can use to validate the signatures over the HCERTs. 
+The resulting list of DSC certificates then provides the acceptable public keys (and the corresponding KIDs) that Verifiers can use to validate the signatures over the HCERTs. Verifiers MUST both fetch and update this list regularly.
 
 Member States may also bilaterally exchange CSCA certificates with a number of other Member States, verify these bilaterally and thus compile their own lists of CSCA and DSC certificates which is specific to that Member State. Verifiers may choose to rely on such a national list.
 
-Such Member State-specific lists are expected to be adapted in the format for their own national setting. As such, the file format of this internal, trusted list may vary, e.g. it can be a plain JWKS like https://github.com/ehn-digital-green-development/hcert-testdata/blob/main/testdata/jwks.json or something specific to the technology used.
+Such Member State-specific lists are expected to be adapted in the format for their own national setting. As such, the file format of this trusted list may vary, e.g. it can be a plain JWKS like https://github.com/ehn-digital-green-development/hcert-testdata/blob/main/testdata/jwks.json or something specific to the technology used.
```

Comment:
Isn't it better to include a portion of the JWKS directly instead of refering to it via a link?

Reply:
Providing a link to a "signed JWKS" instead of "plain JWKS"  makes it easier for verifiers to consume.

Reply:
Good points. I've referenced RFC 7517 section 5 instead, and changed plain to signed.